### DISCUSSION TITLE
LORA + quantization

### DISCUSSION THREAD

**Discussion by SA-j00u** [leejet/stable-diffusion.cpp#245] [Ideas]
auto quantized models with lora leads to unstable/low quality results (especially with lcm lora)
but looks like lora "adding" after model quantization

instead of
can lora be addied along with/before quantization?
to awoid double rounding loss

(i not fully understand how lora exact works and may be wrong
if it's like model merging it's may works
if it's like extra layers maybe i'm wrong)

**Comment by Amin456789** [comment]
indeed, we need lora support for quantized models as well, i got quantized models and wanna use lora with them but it wont let me sadly
